scripts/executions-check.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+install_libpng() {
+	cd ./fuzzers/libfuzzer_libpng && wget https://deac-fra.dl.sourceforge.net/project/libpng/libpng16/1.6.37/libpng-1.6.37.tar.xz 
+	tar -xvf libpng-1.6.37.tar.xz || echo "Failed to download libpng" 
+	cd libpng-1.6.37 && ./configure --enable-shared=no --with-pic=yes --enable-hardware-optimizations=yes && cd ..
+}
+
+build_libpng(){
+	cargo build --release || echo "ERROR: Failed to build libfuzzer_libpng" 
+	
+	cd libpng-1.6.37 && make CC="$(pwd)/../target/release/libafl_cc" CXX="$(pwd)/../target/release/ libafl_cxx" -j "$(nproc)" && cd ..
+}
+
+git_checkout(){
+	git reset --hard HEAD~1		
+}
+
+build_run_fuzzer(){
+	./target/release/libafl_cxx ./harness.cc libpng-1.6.37/.libs/libpng16.a -I libpng-1.6.37/ -o fuzzer_libpng -lz -lm || echo "ERROR: Failed to build the fuzzer"
+
+	./fuzzer_libpng > log.txt &
+	timeout 5s ./fuzzer_libpng >/dev/null 2>&1 &
```

Comment:
let's make it longer, like 60 sec

Reply:
I have changed 5s to 5m in the latest commit.